packages/desktop/src/renderer/engine.ts

```diff
@@ -19,4 +19,10 @@ export const engineStore = engine.getStore()
 export const useEngineStore = <T>(selector?: (state: EngineStateWithActions) => T) => {
   return useStore(engineStore, selector!)
 }
-engine.registerPlugin(new Midi(engine, useEngineStore))
+engine.registerPlugin(new MidiInput(engine))
+
+export const pluginViews = {
+  'midi-input': {
+    inputPanel: MidiInputPanel,
+  },
+}
```

Comment:
Registering plugins in two places now, the engine and `pluginViews` which only `@hedron/desktop` cares about, the engine knowns nothing about this